src/server.ts

```diff
@@ -27,38 +33,60 @@ type Request = http.IncomingMessage & { body?: any };
 type Response = http.ServerResponse;
 
 function isExpress(server): server is IExpressApp {
-  return (typeof server.route === 'function' && typeof server.use === 'function');
+  return typeof server.route === "function" && typeof server.use === "function";
 }
 
 function isPromiseLike<T>(obj): obj is PromiseLike<T> {
-  return (!!obj && typeof obj.then === 'function');
+  return !!obj && typeof obj.then === "function";
 }
 
 function getDateString(d) {
   function pad(n) {
-    return n < 10 ? '0' + n : n;
+    return n < 10 ? "0" + n : n;
```

Comment:
Code style should not be changed, this makes it hard to review and does not provide any benefits. Please, revert styling changes and leave only actual changes.